### PR TITLE
Replace `WMButton` component

### PR DESCRIPTION
**Pull request by dvira**
- Updated all affected files.
- Cleaned up some obsolete/unnecessary code.

### CODE REVIEW

File: src/components/Screen/UsersScreen/LoadMoreWrapper.tsx [hunk 1]
```diff
@@ -41,7 +41,10 @@ function LoadMoreWrapper({
             <span>No results</span>
           )}
           {total_rows > users.length && (
-            <LoadMoreButton onClick={() => fetchUsers(dispatch, envId, from, to, options, users)} />
+            <LoadMoreButton
+              className={classes['load-more-btn']}
+              onClick={() => fetchUsers(dispatch, envId, from, to, options, users)}
```
Comment (dr15) [inline src/components/Screen/UsersScreen/LoadMoreWrapper.tsx:46]: Please change this to a function that is wrapped with a `useCallback()`

**Review by dr15 — CHANGES_REQUESTED** [review]
Please make this tiny tiny change

File: src/components/Screen/UsersScreen/LoadMoreWrapper.tsx (hunk 1 above)
Comment (dvira) [reply src/components/Screen/UsersScreen/LoadMoreWrapper.tsx]: Done

**Review by nunibaranes — APPROVED** [review]
After sanity and bug fixing. 
Looks great!

**Pull request merged**